packages/jaeger-ui/package.json

```diff
@@ -50,6 +50,7 @@
     "@ant-design/compatible": "^5.1.3",
     "@jaegertracing/plexus": "0.2.0",
     "@pyroscope/flamegraph": "0.21.4",
+    "@sentry/browser": "^8.18.0",
```

Comment:
should there be a corresponding change to `yarn.lock`?

Reply:
Yes. I will commit that file as well.